.bacon.yml

```diff
@@ -1,8 +1,97 @@
 test_suites:
-  - name: semgrep
+  - name: "build"
+    abort_commit_on_failure: true
+    criteria: MERGE
+    prereq_test_suite_names:
+      - "lint"
+      - "test"
+      - "typecheck"
+    queue_name: al2023
     script_path: /root/okta/odyssey/scripts
     sort_order: "1"
-    timeout: "10"
-    script_name: semgrep
+    steps:
+      - event: SOURCE
+        action: setup.sh
+        name: Setup Environment
+      - event: COMMAND
+        action: yarn build
+        name: Build Packages
+    timeout: "20"
+
+  - name: publish
+    abort_commit_on_failure: true
+    script_path: /root/okta/odyssey/scripts
+    sort_order: "2"
+    timeout: "15"
+    script_name: publish
+    criteria: MERGE
+    prereq_test_suite_names:
+      - "build"
+    queue_name: al2023
+
+  - name: "code coverage"
+    abort_commit_on_failure: true
+    criteria: MERGE
+    # Playwright only works with certain distros of Linux
+    queue_name: ci-queue-prodJenga-Ubuntu20
+    script_path: /root/okta/odyssey/scripts
+    script_name: code-coverage
+    sort_order: "5"
+    timeout: "20"
```

Comment:
Let's make this optional as it's not part of the current release process and is blocking us right now.